e2e/cypress/support/api_commands.js

```diff
@@ -353,6 +353,49 @@ Cypress.Commands.add('createNewUser', (user = {}, teamIds = []) => {
     });
 });
 
+/**
```

Comment:
Instead of creating new commands, update the existing `createNewUser` and `loginWithNewUser` commands to accept a new param to toggle if we should bypass the tutorial or not.

Reply:
Updated api methods to include bypassTutorial Boolean flag and set default to true; updated this test to set the bypassTutorial flag to false.  Seems to work and other existing tests seem ok.